Makefile

```diff
@@ -38,8 +38,13 @@ docs: ## generate Sphinx HTML documentation, including API docs
 	tox -e docs
 	$(BROWSER) docs/_build/html/index.html
 
+COMMON_CONSTRAINTS_TXT=requirements/common_constraints.txt
+.PHONY: $(COMMON_CONSTRAINTS_TXT)
+$(COMMON_CONSTRAINTS_TXT):
+	wget -O "$(@)" https://raw.githubusercontent.com/edx/edx-lint/master/edx_lint/files/common_constraints.txt || touch "$(@)"
```

Comment:
Oh weird - if the wget fails, we just make an empty one. I assume this is cargo culted and not your own doing? But odd that we wouldn't error out in that case.

Reply:
Yeah, I'm not clear. But I guess the assumption might be some constraint would end up failing (the reason it would have been added to common_constraints to begin with) so the upgrade will still fail?